tests/ui/test_home.py

```diff
@@ -3,21 +3,78 @@
 from pages.desktop.home import Home
 
 
-@pytest.mark.django_db
 @pytest.mark.nondestructive
+@pytest.mark.parametrize('load_fixtures', [('initial_data')], indirect=True)
 def test_there_are_ten_most_popular_extensions(
-        my_base_url, selenium, initial_data):
+        my_base_url, selenium, load_fixtures):
     """Ten most popular add-ons are listed"""
     page = Home(selenium, my_base_url).open()
     assert len(page.most_popular.extensions) == 10
 
 
-@pytest.mark.django_db
 @pytest.mark.nondestructive
+@pytest.mark.parametrize('load_fixtures', [('initial_data')], indirect=True)
 def test_most_popular_extensions_are_sorted_by_users(
-        my_base_url, selenium, initial_data):
+        my_base_url, selenium, load_fixtures):
     """Most popular add-ons are sorted by popularity"""
     page = Home(selenium, my_base_url).open()
     extensions = page.most_popular.extensions
     sorted_by_users = sorted(extensions, key=lambda e: e.users, reverse=True)
     assert sorted_by_users == extensions
+
+
+@pytest.mark.smoke
+@pytest.mark.nondestructive
+@pytest.mark.parametrize('load_fixtures', [('addon')], indirect=True)
+def test_that_clicking_on_addon_name_loads_details_page(
+        my_base_url, selenium, load_fixtures):
+    page = Home(selenium, my_base_url).open()
+    name = page.featured_extensions.extensions[0].name
+    assert name in page.featured_extensions.extensions[0].click().header.title
+
+
+@pytest.mark.smoke
+@pytest.mark.nondestructive
+@pytest.mark.parametrize('load_fixtures', [('themes')], indirect=True)
+def test_that_featured_themes_exist_on_the_home(
+        my_base_url, selenium, load_fixtures):
+    page = Home(selenium, my_base_url).open()
+    assert len(page.featured_themes.themes) >= 6
+
+
+@pytest.mark.nondestructive
+@pytest.mark.parametrize('load_fixtures', [('themes', 'theme')], indirect=True)
+def test_that_clicking_see_all_themes_link_works(
+        my_base_url, selenium, load_fixtures):
+    page = Home(selenium, my_base_url).open()
+    name = page.featured_themes.themes[0].name
+
+    # Match first addon on home page to its name on theme page
+    for theme in page.featured_themes.see_all().featured.themes:
+        if theme.title == name:
+            assert True
+        else:
+            continue
```

Comment:
If `featured.themes` is empty, this test will pass. I would suggest just checking that the length of `featured.themes` on the themes page.